cli-impl/src/test/resources/shared/src/data/either/base.aya

```diff
@@ -0,0 +1,25 @@
+open import relation::binary::path using (= , refl)
+
+open inductive Either (A B : Type) | left A | right B
+
+variable A B : Type
+
+def Either-rec
+  (P : Type)
+  (lrec : A -> P)
+  (rrec : B -> P)
+  (e : Either A B)
+  : P => Either-ind e (fn _e => P) (fn a _p => lrec a) (fn b _p => rrec b)
+
+def Either-ind
+  (e : Either A B)
+  (P : Either A B -> Type)
+  (lrec : Fn (a : A) -> e = left a -> P (left a))
+  (rrec : Fn (b : B) -> e = right b -> P (right b))
```

Comment:
This is probably nonstandard, usually e is the last parameter, and you don't need these e = left a parameters

Reply:
In case the user wants to rewrite something

Reply:
`lrec` technically already assumes `e = left a` IMO